.pipelines/templates/customconfiguration-autoscaling.json

```diff
@@ -17,6 +17,7 @@
       }
     },
     "cluster-autoscaler": {
+      "image": "{CUSTOM_CAS_IMAGE}",
```

Comment:
Please add your own custom config autoscaling json file.

Reply:
I have reverted this change. 
For the actual custom-config autoscaling file, I'll put it into the kubetest2-aks cluster-template directory, 
and point the CUSTOM_CONFIG_PATH to this directory.